### ISSUE TITLE
Feature Request: Add support for scrubbing excess tags

### ISSUE THREAD

**Issue by auranunderscore** [sandreas/tone#67]
A lot of files in my music library have a lot of extraneous tags that aren't useful to me (such as musicbrainz IDs and some iTunes metadata and what have you). The ability to remove all of them in one clean sweep and keep only the tags I want would be really useful. Perhaps as a new input parameter for the remove tagger?

**Comment by sandreas** [comment]
> The ability to remove all of them in one clean sweep and keep only the tags I want would be really useful. Perhaps as a new input parameter for the remove tagger?

This is a great idea. I would probably keep the name `scrub`, because it is a well known term with a set of default tags to `keep`. I have to think this through, because `scrub` intends to do some cleanup, but you have to specify which fields to keep (instead of which fields to remove). Normally scrubbing decribes the process before tagging by an online source (e.g. musicbrainz). 

In the case of `tone` I'd prefer having an option to specify manually, which fields to `keep`. Possibly this could change depending on what media type you are tagging (music, audio book, podcast, etc.).

Thanks for suggesting this. I'll try to get back to you soon.

**Comment by auranunderscore** [comment]
I agree that choosing which fields to `keep` would be very useful given everyone's usecase is different. Thanks for the quick response!!

**Comment by satosti** [comment]
Is this still being planned? Or is there a script that can already be used for this purpose?
